### DISCUSSION TITLE
Create an conda/mamba environment from a pixi.lock file?

### DISCUSSION THREAD

**Discussion by raybellwaves** [prefix-dev/pixi#2460] [Q&A]
I work in a shared jupyterhub and most of my experiment takes place in a notebook and connecting to different kernels (e.g.`mamba env create -f env.yml`) which contains deps and `ipykernel`.

I'm wondering how best to use pixi on the shared jhub when I want to find my kernel for each project.

I'm curious if you can create an conda/mamba environment from a pixi.lock file is or if there is a better way to achieve the same outcome.

Cheers,
Ray

**Comment by abkfenris** [comment]
If you can tweak things at a hub level, there is [pixi-kernel](https://github.com/renan-r-santos/pixi-kernel/issues), otherwise you can use [`pixi project export`](https://pixi.sh/latest/reference/cli/#project-export-conda_environment) to get an environment.yml, or explicit spec.

**Reply by raybellwaves** [reply]
Thanks. I'll take a look at https://github.com/renan-r-santos/pixi-kernel. We're deploying tljh (https://tljh.jupyter.org/en/latest/) so I expect we can add it.

I'll also check out `pixi project export` in the mean time
